api/admin.go

```diff
@@ -356,12 +356,39 @@ func samlMetadata(c *Context, w http.ResponseWriter, r *http.Request) {
 func addCertificate(c *Context, w http.ResponseWriter, r *http.Request) {
 	err := r.ParseMultipartForm(*utils.Cfg.FileSettings.MaxFileSize)
 	if err != nil {
+		l4g.Error(err.Error())
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
 	m := r.MultipartForm
 
+	certificateFile := ""
+
+	if typeArray, ok := m.Value["type"]; !ok {
```

Comment:
This is a breaking change for this endpoint. Can we make type optional, and if it's not included try to infer the type from the filename extension?